ccpp/data/GFS_typedefs.meta

```diff
@@ -3513,19 +3513,12 @@
   dimensions = ()
   type = logical
   intent = in
-[sedi_semi_update]
-  standard_name = flag_for_v_update_in_semi_Lagrangian_sedi
-  long_name = flag for v update in semi Lagrangian sedi of rain
-  units = flag
-  dimensions = ()
-  type = logical
-  intent = in
-[sedi_semi_decfl]
-  standard_name = flag_for_iteration_with_semi_Lagrangian_sedi
-  long_name = flag for interation with semi Lagrangian sedi of rain
-  units = flag
+[decfl]
+  standard_name = deformed_CFL_factor
+  long_name = deformed CFL factor
+  units = count
   dimensions = ()
-  type = logical
+  type = integer
```

Comment:
Note that `intent` does not make sense in the host model metadata - these are defining the variables. The attribute gets ignored. No need to change, I have the next PR and will take care of this (there are other variables in `GFS_typedefs.meta` that do have the `intent` attribute as well).

Reply:
This is fixed in PR https://github.com/NOAA-EMC/fv3atm/pull/422